### PR TITLE
Fix import invalid Credentails

### PR DESCRIPTION
**Pull request by osinakayah**

### CODE REVIEW

File: src/features/credentials/credentials.js [hunk 1]
```diff
@@ -116,6 +116,11 @@ export function useCredentials({onPickFile = pickJSONFile} = {}) {
       return;
     }
 
+    assert(
+      jsonData?.issuer?.hasOwnProperty('id') === true,
```
Comment (mike-parkhill) [inline src/features/credentials/credentials.js:120]: is `id` the only mandatory field, or should we assert others too?

File: src/features/credentials/credentials.test.js [hunk 2]
```diff
@@ -179,10 +179,35 @@ describe('Credentials helpers', () => {
       expect(Credentials.getInstance().remove).toBeCalledWith(1);
     });
 
-    it('expect to add credential', async () => {
-      await act(async () => {
-        await hook.current.onAdd();
+    it('expect to add invalid credential', async () => {
```
[the remark this answers is not in the document]
Comment (osinakayah) [reply src/features/credentials/credentials.test.js]: It is not adding a credential, the test is is making sure that an exception is thrown. The test description has been changed now now

File: src/features/credentials/credentials.js (hunk 1 above)
Comment (osinakayah) [reply src/features/credentials/credentials.js:120]: issuer.id is not, other mandatory fields are  id, @context,type too.  Other wallet documents (DIDsDocument, WalletBackup) has this properties too, except "issuer.id".

File: src/features/credentials/credentials.test.js [hunk 3]
```diff
@@ -179,7 +179,7 @@ describe('Credentials helpers', () => {
       expect(Credentials.getInstance().remove).toBeCalledWith(1);
     });
 
-    it('expect to add invalid credential', async () => {
+    it('expect adding invalid credential to throw exception', async () => {
```
Comment (mike-parkhill) [inline src/features/credentials/credentials.test.js:182]: should we test the various invalid scenarios?

Comment (osinakayah) [reply src/features/credentials/credentials.test.js]: Yes, we can do that.

File: src/features/credentials/credentials.js [hunk 4]
```diff
@@ -89,24 +89,24 @@ export async function processCredential(credential) {
 const validateCredential = credential => {
   assert(
     typeof credential !== 'undefined',
-    translate('credentials.invalid_credential') + 0,
+    translate('credentials.invalid_credential'),
```
Comment (mike-parkhill) [inline src/features/credentials/credentials.js:92]: I kind of liked having some way of knowing what was wrong with the credential based on a code or something. Could we still have that?

Comment (osinakayah) [reply src/features/credentials/credentials.js:92]: Yes, We can have a different  error message based on the mandatory property that is missing.

**Review by mike-parkhill — APPROVED** [review]

**Pull request merged**